core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1688,7 +1692,11 @@ private[spark] object Utils extends Logging {
 
   /**
    * NaN-safe version of `java.lang.Float.compare()` which allows NaN values to be compared
-   * according to semantics where NaN == NaN and NaN &gt; any non-NaN float.
+   * according to semantics where
+   *
+   * {{{
+   * NaN == NaN and NaN > any non-NaN float
+   * }}}.
```

Comment:
- Java
  <img width="830" alt="2016-11-27 3 21 16" src="https://cloud.githubusercontent.com/assets/6477701/20642380/ad9c51cc-b450-11e6-9317-5f8c185e0d05.png">


Scaladoc not found